File: mysql.js
var mysql = require('mysql');

var con = mysql.createConnection({
    host: "localhost",
    user: "owncloud",
    password: "testcloud"
});

con.connect(function(err) {
    if (err) throw err;
    console.log("Connected!");
    con.query("CREATE DATABASE owncloud", function (err, results) {
        if(err) throw err;
        console.log("Database created")
    })
});

File: mysql.py
import mysql.connector

owncloud = mysql.connector.connect(
    host="localhost",
    user="owncloud",
    passwd="testcloud",
)

mycursor = owncloud.cursor()
mycursor.execute("CREATE DATABASE owncloud")
mycursor.execute("GRANT ALL PRIVILEGES ON owncloud.* TO 'owncloud'@'localhost' IDENTIFIED BY 'testcloud'")
mycursor.execute("FLUSH PRIVILEGES")

print('run "python3 python-ocserver/install.py"')